Sprint-3/implement/is-valid-triangle.js

```diff
@@ -33,3 +33,14 @@
 // Then it should return true because the input forms a valid triangle.
 
 // This specification outlines the behavior of the isValidTriangle function for different input scenarios, ensuring it properly checks for invalid side lengths and whether they form a valid triangle according to the Triangle Inequality Theorem.
+function isValidTriangle(a, b, c) {
+    if (a <= 0 || b <= 0 || c <= 0) {
```

Comment:
My thought process while approaching this was since they said we must validate and invalidate our triangle, the only way to do this was through an If statement. Are you saying there's another way of solving this without an If statement?

Reply:
The main point I would like to make is, you should fully understand what you wrote in your code. An interviewer may ask you questions like what I am asking here, and it would reflect poorly on you if you cannot explain your code. 

Thank you for this. I am at the point now in this journey whereby I can read a problem and immediately my mind starts to generate solutions. I have solved problems on code wars, that I didn't grasp their concept. i just knew what I had to do

So my problem is explaining what I am doing or just purely explaining my logic to someone else that's a real struggle but I am grateful for the comment you made. Instead of emphasizing writing the code, will place my emphasis on understanding what is needed, and what my code does and learn to explain my logic to others.